### ISSUE TITLE
reactive fullscreen map

### ISSUE THREAD

**Issue by I3auke** [SymbolixAU/googleway#211]
Hi, I  would expect to create a reactive fullscreen map in shiny via:
```
# ui <- fluidPage(
#   google_mapOutput(outputId = "map", height = "100%")
# )
```
But it is not working (map is not loading at all; only search box is visible).
"height ='800px'" does look better, but it is not reactive (as in; does not respond when window is resized).

thnx. Bauke

**Comment by SymbolixAU** [comment]
Can you post the full shiny code please (you can omit the api key)?

**Comment by I3auke** [comment]
`library(shiny)
library(googleway)
library(shinydashboard)

ui <- fluidPage(google_mapOutput("map",height = '100%'))
latitude<-c(50.814,51.953)
longitude<-c(3.6345, 4.567)
markers <- data.frame(latitude=latitude,longitude=longitude)

server <- function(input, output, session){
  map_key <- "XXXXX"
  output$map <- renderGoogle_map({
    map<-google_map(key = map_key, data = markers,
                    search_box = TRUE, fullscreen_control = TRUE) %>%
      add_markers() 
    map
  })
}
 shinyApp(ui, server)`

**Comment by I3auke** [comment]
```
library(shiny)
library(googleway)
library(shinydashboard)

ui <- fluidPage(google_mapOutput("map",height = '100%'))
latitude<-c(50.814,51.953)
longitude<-c(3.6345, 4.567)
markers <- data.frame(latitude=latitude,longitude=longitude)

server <- function(input, output, session){
  map_key <- "XXXXXX"
  output$map <- renderGoogle_map({
    map<-google_map(key = map_key, data = markers,
                    search_box = TRUE, fullscreen_control = TRUE) %>%
      add_markers()
    map
  })
}

shinyApp(ui, server)
```

**Comment by SymbolixAU** [comment]
Thanks - I can replicate the issue. 

We are getting a `google is not defined` javascript error. 

I'll look into it.

![Screen Shot 2019-11-12 at 8 45 08 am](https://user-images.githubusercontent.com/18344164/68623493-c169f980-0528-11ea-86e3-31f5b7dce17e.png)

**Comment by SymbolixAU** [comment]
> "height ='800px'" does look better, but it is not responsive (it does not respond when window is resized).

But I'm not seeing this behaviour.

**Comment by I3auke** [comment]
found a workaround:
```
ui <- bootstrapPage(
  tags$style(type = "text/css", "html, body {width:100%;height:100%}"),
  google_mapOutput("map",width = "100%", height = "100%"))
```
works  well

**Comment by jimbrig** [comment]
did you ever find a solution for the uncaught reference error?

![image](https://user-images.githubusercontent.com/32652297/116021655-9d17b500-a616-11eb-9486-06a1c398a1ec.png)

**Comment by dcooley** [comment]
nope...

I kind of let it slide given the workaround. Have you tried [the proposed solution](https://github.com/SymbolixAU/googleway/issues/211#issuecomment-553096284) ?

**Comment by jimbrig** [comment]
yes and it caused my map to not render when I set height to "100%". Does it require bootstrapPage?

I am using shinydashboardPlus, and my map is in a held within a box > fluidRow > column width 6.

**Comment by jimbrig** [comment]
scratch that.

simply changing height to 400px worked. 

Pretty weird given that the default arguments already use 100% and 400px for width and height:


```r
google_mapOutput
function (outputId, width = "100%", height = "400px") 
{
    htmlwidgets::shinyWidgetOutput(outputId, "google_map", 
        width, height, package = "googleway")
}
```

This is all I changed:

![image](https://user-images.githubusercontent.com/32652297/116022878-12848500-a619-11eb-8be4-1b0eb9aed869.png)
